app/models/user.rb

```diff
@@ -2,4 +2,7 @@ class User < ApplicationRecord
   include GDS::SSO::User
 
   serialize :permissions, Array
+  def is_super_admin?
+    email.downcase.include?("@digital.cabinet-office.gov.uk")
```

Comment:
```suggestion
    email.downcase.include?("@digital.cabinet-office.gov.uk")
```

This is a temporary "phase 1" implementation of how we identify super admins.

We would use this is a pundit policy to restrict who can access the list of users and be able to add/edit them.

Once we had setup  super admins initially in prod then we can update this to use the `super_admin_user` value
or even just the signons permission (which I dont think we need to use at all)